docs/pages/preview/eas-update/getting-started.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Getting started with EAS Update
+---
+
+Setting up EAS Update allows you to push critical bug fixes and improvements that your users need right away. Setting update EAS Update requires:
+
+- Installing expo-cli and eas-cli
+- Creating an Expo account
+- Creating a project
+- Configuring your project's app config
+- Configuring expo-asset and expo-updates
+- Creating a build for the project
+- Publishing an update
+
+## Installing expo-cli and eas-cli
+
+1. Install expo-cli and eas-cli globally:
+
+   ```bash
+   npm install --global eas-cli expo-cli
+   ```
+
+## Create an Expo account
+
+1. Create an account at [https://expo.dev/signup](https://expo.dev/signup)
+2. Then, log in with eas-cli:
+
+   ```bash
+   eas login
+   ```
+
+3. After logging in, you can verify the logged in account with `eas whoami`.
+
+## Create a project
+
+1. Create a project by running:
+
+   ```bash
+   expo init
+   ```
+
+2. Select "Managed workflow > blank".
+
+## Configuring your project's app config
+
+1. We'll need to register our app with EAS and add our project's ID to **app.json**. Run:
+
+   ```bash
+   eas init
+   ```
+
+   This command will add an object with your project's `projectId` in **app.json**. Copy this ID for the next step.
+
+2. Next, in **app.json**, add an `expo.updates.url` property with the following URL, replacing the `your-project-id` with the `projectId` added in the previous step.
+
+   ```json
+   {
+     "expo": {
+       "updates": {
+         "url": "https://u.expo.dev/your-project-id"
+       }
+     }
+   }
+   ```
+
+   There is also a `fallbackToCacheTimeout` property. If you'd like your app to try to load new updates when a user opens the app, set this to something other than zero, like `3000` (3 seconds). A value of `3000` would mean that your app will try and download a new update for up to 3 seconds before loading a cached version of your app. If the app is able to download the update within 3 seconds, your users will see the changes in the newest update immediately.
+
+3. Next, set a `expo.runtimeVersion` property in the project's **app.json** file. Let's use `"1.0.0"` as the runtime version's value:
+
+   ```json
+   {
+     "expo": {
+       "runtimeVersion": "1.0.0",
```

Comment:
```suggestion
       "runtimeVersion": {"plugin": "nativeBuild"},
```
maybe something we want to guide people to until they are sophisticated enough to handle managing it on their own?